### ISSUE TITLE
Deleting element leaves node behind

### ISSUE THREAD

**Issue by TimeGentleman** [bigjko/cwine#11]
Deleting an element leaves its node and dotted line behind. This screws up the editor scrolling. The orphan nodes disappear on a reload and this also fixes the scrolling issue.

**Comment by bigjko** [comment]
Yeah, I need to fix this later tonight. Removing and Adding panels is completely borked in this latest version. It also resets the new variables (width, height, comic width, etc), which is a bummer. So hang on just a bit, I will fix it!
